api/core/model_runtime/model_providers/bedrock/llm/llm.py

```diff
@@ -48,6 +48,28 @@
 
 class BedrockLargeLanguageModel(LargeLanguageModel):
 
+    # please refer to the documentation: https://docs.aws.amazon.com/bedrock/latest/userguide/conversation-inference.html
+    # TODO There is invoke issue: context limit on Cohere Model, will add them after fixed.
+    CONVERSE_API_ENABLED_MODEL_INFO=[
```

Comment:
I think we need to move this new features `support_system_prompts` & `support_system_prompts` to yaml files. Thoughts? @laipz8200 @takatost

Reply:
Yes, i think so, but didn't know if there is a yaml file format standard?

Reply:
> I think we need to move these new features `support_system_prompts` & `support_system_prompts` to yaml files. Thoughts? @laipz8200 @takatost

Just checked in model_entities.py, there is no key refer to system prompts, would you add it later? or what should I do next?